tests/test_main.py

```diff
@@ -339,3 +339,56 @@ def test_main_noxfile_options_sessions():
         # Verify that the config looks correct.
         config = honor_list_request.call_args[1]["global_config"]
         assert config.sessions == ["test"]
+
+
+@pytest.mark.parametrize(("isatty_value", "expected"), [(True, True), (False, False)])
+def test_main_color_from_istty(isatty_value, expected):
+    sys.argv = [sys.executable]
+    with mock.patch("nox.workflow.execute") as execute:
+        execute.return_value = 0
+        with mock.patch("sys.stderr.isatty") as istty:
```

Comment:
ISTR you prefer naming the mock object the same as the thing being mocked, in which case `istty` -> `isatty`

Reply:
Great catch